src/main/java/com/urswolfer/gerrit/client/rest/http/groups/GroupApiRestClient.java

```diff
@@ -0,0 +1,253 @@
+package com.urswolfer.gerrit.client.rest.http.groups;
+
+import com.google.gerrit.extensions.api.groups.GroupApi;
+import com.google.gerrit.extensions.common.AccountInfo;
+import com.google.gerrit.extensions.common.GroupAuditEventInfo;
+import com.google.gerrit.extensions.common.GroupInfo;
+import com.google.gerrit.extensions.common.GroupOptionsInfo;
+import com.google.gerrit.extensions.restapi.BinaryResult;
+import com.google.gerrit.extensions.restapi.NotImplementedException;
+import com.google.gerrit.extensions.restapi.RestApiException;
+import com.google.gerrit.extensions.restapi.Url;
+import com.google.gson.JsonElement;
+import com.urswolfer.gerrit.client.rest.http.GerritRestClient;
+import com.urswolfer.gerrit.client.rest.http.util.BinaryResultUtils;
+import org.apache.http.HttpResponse;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URLEncoder;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.SortedMap;
+
+
+/**
+ * Extend the Gerrit implementation in order to add custom functionality
+ * that is not currently supported in the API.
+ *
+ * @author Shawn Stafford
+ */
+public class GroupApiRestClient implements GroupApi {
+
+    /** Base REST URL for managing Group data */
+    public static final String BASE_URL = "/groups";
+
+
```

Comment:
BASE_URL is referenced from another class (GroupsRestClient) because the URL being constructed in the create() method needs to it.  I'll make the variable private, but will need to add a method in GroupApiRestClient for accessing the information.